crates/env/src/engine/off_chain/test_api.rs

```diff
@@ -420,11 +421,18 @@ pub fn assert_contract_termination<T, F>(
 {
     let value_any = ::std::panic::catch_unwind(should_terminate)
         .expect_err("contract did not terminate");
-    let encoded_input: &Vec<u8> = value_any
-        .downcast_ref::<Vec<u8>>()
+    let encoded_input = value_any
+        .downcast_ref::<String>()
         .expect("panic object can not be cast");
+    let deserialized_vec = encoded_input
+        .replace("[", "")
+        .replace("]", "")
+        .split(", ")
+        .map(|s| u8::from_str(s).expect("u8 cannot be extracted from str"))
+        .collect::<Vec<u8>>();
```

Comment:
I dislike this hacky way of deserializing, but AFAIK it's the simplest way of getting the inverse of a `panic!("{:?}", Vec<u8>`). We can get rid of this as soon as `panic_any` is stablized, which should be with the next stable release soonish.

Reply:
Can you create an issue for that so that we do not forget?